web/unit_tests/cases/events.js

```diff
@@ -48,9 +47,11 @@ describe('Event Management', function() {
     if(focusEvent) {
       ele.dispatchEvent(focusEvent);
     }
+
+    assert.isNull(ele.onchange);
```

Comment:
I see what this is doing but it's a bit indirect. Perhaps a comment or assertion message would help.